web-api/src/persistence/dynamo/trialSessions/getTrialSessionWorkingCopy.ts

```diff
@@ -17,11 +39,28 @@ export const getTrialSessionWorkingCopy = ({
   applicationContext: IApplicationContext;
   trialSessionId: string;
   userId: string;
-}) =>
+}): TrialSessionWorkingCopy =>
   get({
     Key: {
       pk: `trial-session-working-copy|${trialSessionId}`,
       sk: `user|${userId}`,
     },
     applicationContext,
   });
+
+export const getBulkTrialSessionWorkingCopies = async ({
```

Comment:
I think the pattern on the app is typically to split these functions out into separate files, e.g., `getBulkTrialSessionWorkingCopy.ts` and `getBulkTrialSessionWorkingCopy.test.ts`.